src/cleanIrcMessage.js

```diff
@@ -0,0 +1,4 @@
+"use strict";
+
+// TODO: This does not strip hex based colours
+module.exports = (message) => message.replace(/\x02|\x1D|\x1F|\x16|\x0F|\x03(?:[0-9]{1,2}(?:,[0-9]{1,2})?)?/g, "").trim();
```

Comment:
This fails on Node 8:

https://travis-ci.org/thelounge/lounge/jobs/280780940#L2204

Reply:
I sure am glad we run production builds in our CI!

According to https://stackoverflow.com/questions/43888474/unexpected-token-operator-from-uglifyjs, Webpack relies on a version that doesn't support ES6 yet, so hopefully by the time this PR gets merged, they will be done (?).
I'm really surprised why this arrives only now though, since that's not the first fat-arrow function we use on the server...

Reply:
That was because `src` folder wasn't included in babel transpiling.

Reply:
Do we want to add the `src` though? I don't know, I feel odd about that. We have client code we use on the server and now server code we use on the client :/

Also does that mean more gets added to the client bundle? I compared and it doesn't seem to any extra kb heavier.

Reply:
I could move this file to client (and reference it on the server like we already do with findLinks) instead, this will at least keep the consistency in this regard.

Reply:
That seems doing yeah

Reply:
Fine*

Damn, 1.5month to notice my typo...